fcrepo-exts/migration-utils
# Validate fedora 3 checksums for datastream content

Validate fedora 3 checksums for datastream content
* * *

**JIRA Ticket**: (https://jira.lyrasis.org/browse/FCREPO-3642)

https://wiki.lyrasis.org/display/FF/2021-02-18-+Fedora+Tech+Meeting

# What does this Pull Request do?
Fedora 3 often has checksums for its datastream content. When migrating content over to OCFL, we should verify that the datastream content matches its checksums. This PR adds the feature, and also adds an option to disable checksum validation if desired.

# How should this be tested?
There are a couple automated tests, but to test manually: take a fedora3 object, manually change the checksum to be invalid, then run the migration, and it should fail.

# Additional Notes:
Documentation will need to be updated.

# Interested parties
@fcrepo/committers
